src/types/index.ts

```diff
@@ -639,9 +639,9 @@ export type MoveUint32Type = number;
 export type MoveUint64Type = string;
 export type MoveUint128Type = string;
 export type MoveUint256Type = string;
-export type MoveAddressType = `0x${string}`;
-export type MoveObjectType = `0x${string}`;
-export type MoveStructType = `0x${string}::${string}::${string}`;
+export type MoveAddressType = string;
+export type MoveObjectType = string;
+export type MoveStructType = `${string}::${string}::${string}`;
```

Comment:
I suspect user (like me) will dig in and check what these types are. 

Shall we add a note here for the user to know, that there should be `0x` prefix?

Reply:
We can probably document that on each type